mockgcp/Makefile

```diff
@@ -31,8 +18,13 @@ gen-proto: tools
 	git clone https://github.com/googleapis/googleapis.git third_party/googleapis || (cd third_party/googleapis && git reset --hard origin/master && git pull)
 	mkdir -p ./generated
 
+	./apply-proto-patches.sh
```

Comment:
If we want to make the apply-proto-patches a permanent solution (I think we do), then maybe call out that this is an one-off run since the patch always appends the proto?